app/utilities/decimal_places.py

```diff
@@ -0,0 +1,94 @@
+from decimal import Decimal
+from typing import Literal, TypeAlias
+
+from babel import Locale, numbers, units
+
+UnitLengthType: TypeAlias = Literal["short", "long", "narrow"]
+
+
+def custom_format_decimal(value: int | Decimal | float, locale: Locale | str) -> str:
```

Comment:
Need unit tests for all the public methods in this file

Reply:
Added tests for these methods now 👍